doc/reference/reference_lua/compat/box_info_cluster_meaning.rst

```diff
@@ -0,0 +1,72 @@
+.. _compat-option-box-info-cluster:
+
+Meaning of box.info.cluster
+===========================
+
+Option: ``box_info_cluster_meaning``
+
+Starting from version 3.0, the :ref:`box_info_cluster` table stores the information
+about the entire cluster. In earlier versions, it stored only the current replica set
+information. The ``box_info_cluster_meaning`` compat option allows to define the
+the scope of ``box.info.cluster``: the entire cluster or a single replica set.
+
+Old and new behavior
+--------------------
+
+New behavior: ``box.info.cluster`` displays information about the entire
+cluster with all its replica sets.
+
+..  code-block:: tarantoolsession
+
+    tarantool> compat.box_info_cluster_meaning = 'new'
+    ---
+    ...
+
+    tarantool> box.info.cluster
+    ---
+    - <cluster information>
+    ...
+
+    tarantool> box.info.replicaset
+    ---
+    - uuid: <replicaset uuid>
+    - <... other attributes of the replicaset>
+    ...
```

Comment:
I think it would be convenient to have an example of a real output (preferably with persistent names set).

We say 'entire cluster information' and 'replicaset information' above, but it is very abstract. As a user I can expect a list of instances, their metrics, configuration and so on.

In fact, we have `box.info.replicaset.{uuid,name}` and `box.info.cluster.name`. That's all the fields.